app/controllers/courses_controller.rb

```diff
@@ -0,0 +1,39 @@
+class CoursesController < ApplicationController
```

Comment:
Ah OK.
Btw, I also need to put these 3 guys in a folder called courses(under controllers folder), right ? 
> module Courses; end
> class Courses::Controllers < ApplicationController; end
> class Courses::CoursesController < Controllers; end